lua/octo/gh/init.lua

```diff
@@ -56,8 +56,8 @@ function M.get_user_name(remote_hostname)
   job:sync()
   local stderr = table.concat(job:stderr_result(), "\n")
   local stdout = table.concat(job:result(), "\n")
-  local name_err = string.match(stderr, "Logged in to [^%s]+ as ([^%s]+)")
-  local name_out = string.match(stdout, "Logged in to [^%s]+ as ([^%s]+)")
+  local name_err = string.match(stderr, "Logged in to [^%s]+ account ([^%s]+)")
```

Comment:
Thanks, can you make it try the old regexp if the new one fails for backwards compatibility?

Reply:
resolved 👍